backend/settings.js

```diff
@@ -18,7 +18,8 @@ const DEFAULT_SETTINGS = {
     PLOTLY_API_DOMAIN: 'api.plot.ly',
 
     // TODO - Should this be configurable or should it just be a constant?
-    CONNECTOR_HTTPS_DOMAIN: 'connector.plot.ly',
+    CONNECTOR_HOST_INFO: {host: '', lastUpdated: ''},
```

Comment:
refering to another comment above, it might be simpler if we just flattened this out into two settings:
```
CONNECTOR_HOST
CONNECTOR_HOST_LAST_UPDATED
```

otherwise, in the YAML it'll get saved as 
```
CONNECTOR_HOST_INFO: {"host": "...", "lastUpdated: ""}
```
because we're not setting the `infinity` flag because it's buggy: https://github.com/plotly/plotly-database-connector/pull/132/files#diff-6f3d7edbdf97973db754a3eff395e817L150